planning_launch/config/obstacle_stop_planner.yaml

```diff
@@ -0,0 +1,15 @@
+/**:
+  ros__parameters:
+    # stop planner
+    stop_margin: 5.0 # stop margin distance from obstacle on the path [m]
+    min_behavior_stop_margin: 5.0 # stop margin distance when any other stop point is inserted in stop margin [m]
+    step_length: 1.0 # step length for pointcloud search range [m]
+    extend_distance: 0.0 # extend trajectory to consider after goal obstacle in the extend_distance
+    expand_stop_range: 0.0 # margin of vehicle footprint [m]
+
+    # slow down planner
```

Comment:
Refer this.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/b0ba4f052ed836f5bc69e87d7f3ca83fec37f8e2/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/config/obstacle_stop_planner.yaml#L10

Reply:
fixed